build_windows.ps1

```diff
@@ -0,0 +1,20 @@
+function ErrorOnExeFailure {
+  if (-not $?)
+  {
+    throw 'Last EXE Call Failed!'
+  }
+}
+
+Write-Host "Fetching git tags"
+git fetch --tags
+ErrorOnExeFailure
+
+Write-Host "Configuring with CMake"
+mkdir out
+cd out
+cmake .. -G "Visual Studio 15 2017" -DTARGET_CPU=x86
```

Comment:
Are we building web rtc stuff as 32 bit (as this code suggests) on purpose?  Hmm, maybe it is because we build the Windows Solstice server 32-bit at this time!!!  OK, in the future when we change the Windows Server to be 64-bit, do we see any issues with doing WebRTC all in 64 bit too?

Reply:
That's correct. Windows Server is 32bit. No problem building this for Win64 in the future, as far as I can tell. Super easy from a CI perspective!

Reply:
Sounds good!